ctx.go

```diff
@@ -1415,3 +1415,8 @@ func (c *Ctx) IsFromLocal() bool {
 	}
 	return c.isLocalHost(ips[0])
 }
+
+// Protocol returns the request's HTTP protocol: HTTP/1.1,  HTTP/1.0
+func (c *Ctx) Protocol() string {
+	return string(c.fasthttp.Request.Header.Protocol())
```

Comment:
@abhi12299 pls use this https://github.com/gofiber/fiber/blob/master/utils/convert.go#L16

Reply:
Done. Check: [9667849](https://github.com/gofiber/fiber/pull/1746/commits/96678497eba1722057e63d5b0082df9fbaf299c6)